daffodil-lib/src/main/scala/org/apache/daffodil/util/Coroutines.scala

```diff
@@ -98,6 +103,21 @@
    protected def run(): Unit
  }
 
+ /**
```

Comment:
Another minor tweak to coroutines. This pattern comes up everywhere coroutines are used.